recipes/jasper/all/conanfile.py

```diff
@@ -1,18 +1,21 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, rmdir, rm, save, export_conandata_patches, apply_conandata_patches
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv
 import os
 import textwrap
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.52.0"
 
 
 class JasperConan(ConanFile):
     name = "jasper"
-    license = "JasPer License Version 2.0"
+    license = "LicenseRef-LICENSE"
```

Comment:
Can the license here be the SPDX identifier `JasPer-2.0`, documented [here](https://spdx.org/licenses/JasPer-2.0.html)?